processor/src/decoder/tests.rs

```diff
@@ -1000,6 +1003,13 @@ fn call_block() {
         assert_eq!(program_hash, get_hasher_state1(&dec_trace, i));
     }
 
+    // --- check the scf column -------------------------------------------------------------------
+
+    // since no syscalls were made, values in the syscall flag column should be all ZEROs
+    for i in 0..trace_len {
+        assert_eq!(dec_trace[IN_SYSCALL_COL_IDX][i], ZERO);
```

Comment:
nit: one alternative way to describe this check is to use assert instead of a loop. that way, you have a single statement, even tho the compiler might compare the slice with a loop (but then its up to him)

Something like `assert_eq!(&dec_trace[IN_SYSCALL_COL_IDX][..trace_len], vec![ZERO; trace_len])`

Reply:
Fixed!